src/contracts/IChain.sol

```diff
@@ -5,45 +5,52 @@ pragma abicoder v2;
 interface IChain {
   struct Block {
     bytes32 root;
-    address minter;
```

Comment:
In https://github.com/umbrella-network/spear-lib I added  contracts as a submodule. Much neater solution because there's not need to keep ABI and sol files.

Reply:
I will check that.. I never use it, but I heard about it... looks like about time to learn it

Reply:
I created a ticket for that.